python_src/src/lamp_py/postgres/postgres_utils.py

```diff
@@ -17,13 +16,8 @@
 from .postgres_schema import (
     MetadataLog,
     SqlBase,
-    VehicleEventMetrics,
-    VehicleEvents,
-    VehicleTrips,
 )
 
-HERE = os.path.dirname(os.path.abspath(__file__))
```

Comment:
no longer need a path to this file. it was used to derive a relative path to the seed file.